server/services/getCompany.js

```diff
@@ -0,0 +1,12 @@
+const scrapeEbay = require("./ebayScraper");
+const scrapeAmazon = require("./scraperAmazon");
+const scrapeCraigslist = require("./scraperCraigslist");
+
+const getCompany = (url) => {
+  if (url.includes("amazon")) return [scrapeAmazon, "amazon"];
+  if (url.includes("ebay")) return [scrapeEbay, "ebay"];
+  if (url.includes("craigslist")) return [scrapeCraigslist, "craigslist"];
```

Comment:
we can definitely make this a bit more robust so that something like `https://www.ebay.ca/itm/Amazon-Fire-TV-Stick-4K-Alexa-Voice-Remote-Streaming-Latest-2019-Version/154026065428?epid=7035873780&hash=item23dcab3e14:g:gZsAAOSwMK5fIfb7` will also work. Perhaps splitting by checking right after the `https://` and `www.`

Reply:
yup, that's a good idea and I'll fix it